src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("Forecast Details", () => {
+  const validProps = {
+    forecast: {
+      date: 1111111,
+      temperature: {
+        min: 12,
+        max: 22,
+      },
+      wind: {
+        speed: 13,
+        direction: "s",
+      },
+      humidity: 30,
+    },
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastDetails
+        forecast={validProps.forecast}
+        date={validProps.forecast.date}
+        humidity={validProps.forecast.humidity}
+        temperature={validProps.forecast.temperature}
+        wind={validProps.forecast.wind}
+      />
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
Could some extra tests be added here? Since we are controlling every value that goes into the component, we could validate that the right text is being displayed for each of the props passed in, for example for humidity:
```
expect(
  getByText(`Humidity: ${validProps.forecast.humidity}`)
).toBeInTheDocument();
```